Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -406,6 +402,23 @@ TextureCache::TCacheEntryBase* TextureCache::Load(const u32 stage)
 		++iter;
 	}
 
+	if (unconverted_copy != textures.end())
+	{
+		// Perform palette decoding.
+		TCacheEntryBase *entry = unconverted_copy->second;
+		TCacheEntryBase *decoded_entry = AllocateTexture(entry->config);
```

Comment:
Directly copying the configuration of the input is fragile, but adding a check seems sort of awkward. Changed to explicitly set up the configuration.